django_large_image/templates/django_large_image/_include/colors.html

```diff
@@ -18,15 +18,90 @@
     @click="controls = ! controls"
   >
     Controls
-    <i class="ri-equalizer-line text-lg font-normal" x-show="!controls"></i>
-    <i class="ri-close-line text-lg font-normal" x-show="controls"></i>
+    <i class="ri-equalizer-line" x-show="!controls"></i>
+    <i class="ri-close-line" x-show="controls"></i>
   </button>
 
-  <div class="absolute bottom-full pb-3 right-0" x-show="controls">
-    <div class="card bg-gray-100 bg-opacity-[.90] border border-gray-300 min-w-max !rounded-lg shadow-lg">
-      <div class="card-body">
-        <div class="grid grid-cols-2 gap-8">
-          <div class="controls flex flex-col gap-6">
+  <div
+    class="absolute bottom-full pb-3 right-0"
+    x-show="controls"
+  >
+    <div class="card bg-slate-100 bg-opacity-[.85] min-w-max !rounded-lg shadow-lg">
+      <div
+        class="card-body !p-0"
+        x-data="{ customizeDisplay: false }"
+      >
+        <div
+          class="grid gap-4"
+          :class="{ 'grid-cols-2' : customizeDisplay }"
+        >
+          <div
+            x-show="customizeDisplay"
+            x-data="{ configure: false }"
+            class="row-span-full w-56"
+          >
```

Comment:
I've created a variable `configure` which is set to false. When and of the "band" list items is clicked, it sets the variable to true, showing the configure block.

This is most likely going to need to get blown up, I'm just unsure of the logic to make it work as intended. The way it works now is when you click a "band" list item, ALL of the band list items become active, and a static configuration box shows up below. The intended functionality is only the clicked band becomes active, and the configuration box that shows up is specific to the band item that was clicked.